docs/configuration/http_filters/grpc_json_transcoding_filter.rst

```diff
@@ -0,0 +1,19 @@
+.. _config_http_filters_grpc_json_transcoding:
+
+gRPC-JSON transcoding filter
+============================
+
+gRPC :ref:`architecture overview <arch_overview_grpc>`.
+
+This is a filter which enables the bridging of a JSON-REST client to a gRPC server.
```

Comment:
I don't think this is enough documentation for someone to actually use this filter effectively. Do you have any other documentation somewhere that we can link to? Otherwise, can we add some more documentation? And maybe do a follow up at some point with a full docker example? I think there should be some overview discussion of this functionality on this page also: https://lyft.github.io/envoy/docs/intro/arch_overview/grpc.html